src/components/BlogPost/blogs.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import weekendfly from '../../images/weekendfly.jpg';
+import redclay from '../../images/redclay.jpg';
+import nightfamily from '../../images/nightfamily.jpg';
+
+const blogList = [
+  {
+    name: 'The Last Shadow Puppets – The Age of the Understatement',
+    blogSideDate: 'September 2, 2017',
+    blogSideTags:
+      '60s, Alex Turner, Arctic Monkeys, Bond, James Bond, rock, The Last Shadow Puppets',
+    description:
```

Comment:
If you want to bold something or use html instead of a string for description, you can do...

```description: (<p>this should be <b>Bold</b> right there </p>),```